benchmarks/mo_create_study.py

```diff
@@ -0,0 +1,44 @@
+import json
```

Comment:
The `benchmarks` directory becomes a bit complicated. How about creating a `benchmarks/kurobako` directory and moving `mo_create_study.py` into this?

Reply:
I agree your comments. I move `mo_create_study.py` into `into benchmarks/kurobako`.